### PR TITLE
footer should exist on tablet widths (bug 1129031)

### PR DESCRIPTION
**Pull request by spasovski**

### CODE REVIEW

File: src/media/css/newsletter.styl [hunk 1]
```diff
@@ -137,5 +143,11 @@
     .newsletter-signup-button,
     .newsletter-cancel-button {
         width: auto;
+        margin: 0;
+    }
+    .newsletter-footer {
+        border-bottom: 1px solid $cloud-gray;
```
Comment (ngokevin) [inline src/media/css/newsletter.styl]: let's give it the greyscale palette while we're at it

File: src/templates/footer.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div id="newsletter-footer" class="c">
+<div id="newsletter-footer" class="newsletter-footer c">
```
Comment (ngokevin) [inline src/templates/footer.html]: still need the id?

**Comment by ngokevin** [comment]
you could add a quick test in index.js using helpers.tabletTest test wrapper r+wc

File: src/templates/footer.html (hunk 2 above)
Comment (spasovski) [inline src/templates/footer.html]: Yea the JS still references it.

**Pull request merged**